### DISCUSSION TITLE
Oculus Rift S support

### DISCUSSION THREAD

**Discussion by tolgayilmaz86** [retroluxfilm/reshade-vrtoolkit#2] [Q&A]
Is Oculus Rift S supported as well with the current listed games if I open them through SteamVr ? Or what can I do as a workaround?

**Comment by retroluxfilm** [comment]
Yes if you use the steamVR mode it works as well with the oculus. I heard that there is a beta version that improved the peformance impact when using steamVR. I dont have a oculus but some others might help you out.

**Reply by tolgayilmaz86** [reply]
Finally tried it on oculus rift s with Automobilista 2 but I noticed no change on VR lens instead on monitor. Maybe I've skipped a step but yet not enough tutorial for Oculus rift i guess. Will wait for Oculus support :)

**Reply by retroluxfilm** [reply]
I think you can use the free tool https://store.steampowered.com/app/1009850/OVR_Advanced_Settings/ to force steamVR API instead of the oculus one. Give it a try and let me know if it worked.

**Reply by tolgayilmaz86** [reply]
I've already tried with that but; did not see the icon (labeled as 1) as in the image in configuration page. So I've made my changes on reshade through monitor user interface.

**Reply by retroluxfilm** [reply]
@tolgayilmaz86 did you get it working on AMS2 with your oculus? Would be nice for others having the same issue to preset a solution. :)

**Reply by tolgayilmaz86** [reply]
Unfortunately could not managed to work them both together.

**Reply by retroluxfilm** [reply]
You could try to use the new version and name the 64 bit dll to dinput8.dll and start AMS2 with this `-vr -hmd=SteamVR` commandline parameters. Make sure to enter the racetrack and be into the cockpit for reshade to properly activate. You can sent me your dinput8.log that is created alongside the dll, that I can check it if is started and hooked correctly.

**Reply by tolgayilmaz86** [reply]
thanks  I will try once I set my pc up again.
